test/features/deploy_service_resources.feature

```diff
@@ -0,0 +1,33 @@
+@resoures
+Feature: Deploy the service by configuring the resource requests and limits
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+
+  @runtimerequests @runtimelimits
+  Scenario Outline: Setting runtime resource "requests" and "limits"
+    Given Clone Kogito examples into local directory
+    And Local example service "ruleunit-quarkus-example" is built by Maven
+
+    When Create service "ruleunit-quarkus-example" with runtime requests <requests> and limits <limits>
+    And BuildConfig "ruleunit-quarkus-example-binary" is created after 1 minutes
+    And Start build with name "ruleunit-quarkus-example-binary" from local example service path "ruleunit-quarkus-example/target"
+
+    Then Kogito application "ruleunit-quarkus-example" has 1 pods running within 10 minutes
+    And Kogito application "ruleunit-quarkus-example" has pods with runtime requests <requests> and limits <limits> within 2 minutes
```

Comment:
same here `with runtime resources:`  and datatable

Reply:
done